nightswatcher/nightswatcher.py

```diff
@@ -196,7 +196,7 @@ def extract_build(artifact_zip, build):
         f.close()
 
         if stable is True:
-            shutil.copy2(link_file, link_nightly)
+            shutil.copy2(link_file, link_file_nightly)
```

Comment:
This line actually isn't quite right (or incomplete) because it refuses to overwrite. I haven't investigated the solution yet. Probably just `os.remove` if `os.path.exists`. Anyway, it just means the development channel is currently missing the stable release, nothing too serious.

Reply:
Err, my H2O did offer me the stable earlier tonight? It even worked and everything ^^.

Reply:
Yeah, this is only for Android/AppImage. ;-)